charts/camunda-platform/values.yaml

```diff
@@ -872,6 +872,29 @@ identity:
   keycloak:
     # Keycloak.enabled is used incorporate with "global.identity.keycloak" to use your own Keycloak instead of the one comes with Camunda Platform Helm chart.
     enabled: true
+
+    # NOTE: Since Helm v3 (latest checked 3.10.x) doesn't merge lists with custom values files, then you will need to
+    # add this to your own values file if you override any of "extraVolumes", "initContainers", or "extraVolumeMounts".
+    extraVolumes:
+    - name: camunda-theme
+      emptyDir:
+        sizeLimit: 10Mi
+
+    initContainers:
+    - name: copy-camunda-theme
+      image: >-
+        {{- $identityImage := (dict "base" .Values.global "overlay" .Values.global.identity) -}}
+        {{- include "camundaPlatform.image" $identityImage }}
```

Comment:
:thinking: :question: why we have here the separate var and not like above in one include?

Reply:
because I want to set the max line length to 120 :grinning: 
so I split any new long lines till we have the linting part of the CI pipelines.

Reply:
ah alright I felt it will have a different out come :shrug: